static/index.html

```diff
@@ -520,13 +520,13 @@ <h3 class="mb-5">Other switches</h3>
           },
           logout: function () {
             this.auth0.logout({
-              returnTo: 'http://localhost:3000/'
+              returnTo: window.location.origin
             });
           },
           logoutNoClient: function () {
             this.auth0.logout({
               client_id: null,
-              returnTo: 'http://localhost:3000/'
+              returnTo: window.location.origin
```

Comment:
Integration test fail because of the missing trailing `/`.

Reply:
Yeah I noticed that, will push a fix soon.